bentoml/_internal/cli/bento_management.py

```diff
@@ -262,8 +263,8 @@ def push(bento_tag: str, force: bool):
     @click.argument("build_ctx", type=click.Path(), default=".")
     @click.option("-f", "--bentofile", type=click.STRING, default="bentofile.yaml")
     @click.option("--version", type=click.STRING, default=None)
-    def build(build_ctx, bentofile, version):
+    def build(build_ctx: str, bentofile: str, version: str) -> None:
         if sys.path[0] != build_ctx:
             sys.path.insert(0, build_ctx)
 
-        build_bentofile(bentofile, build_ctx=build_ctx, version=version)
+        return build_bentofile(bentofile, build_ctx=build_ctx, version=version)
```

Comment:
just a nitpick, maybe this is a python convention, but is the "return" intended on a method which returns -> None?

Reply:
Usually, if a function returns None, we shouldn't return. In python this is acceptable:
```python
def func():
    test = "hello"
    return test

func()
```
However, this is a CLI command, users wouldn't be able to access the return value. 

`return` here is a bit of a hack. Since this is a click command, we are wrapping every click function through `BentoMLCommandGroup` under `click_utils.py`. You can check under `track_usage` commands is that we actually use this return value for tracking.

Reply:
Shouldn't we just annotate the correct return type, then?